.appveyor.yml

```diff
@@ -10,18 +10,15 @@ clone_folder: c:\mill
 environment:
   matrix:
   - COMPILER: default
-    JAVA_HOME: C:\Program Files\Java\jdk9
+    JAVA_HOME: C:\Program Files\Java\jdk1.8.0
   - COMPILER: cygwin
     CYGWIN_DIR: cygwin64
     JAVA_HOME: C:\Program Files\Java\jdk9
   - COMPILER: msys2
     MSYS2_ARCH: x86_64
     MSYS2_DIR: msys64
     MSYSTEM: MINGW64
-    JAVA_HOME: C:\Program Files\Java\jdk1.8.0
-
-cache:
-  - '%LOCALAPPDATA%\Coursier\cache -> build.sc'
```

Comment:
Also, disabled caching to see if that helps with weird `MatchError`s due to `wrong checksum`, e.g., https://ci.appveyor.com/project/lihaoyi/mill/build/1.0.475/job/3n4rcqtqpeji55ur